platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpLogoutActionProvider.java

```diff
@@ -56,7 +56,9 @@
             try {
 
                 @SuppressWarnings("unchecked")
-                String nameId = SubjectUtils.getName((Subject) ((Map) realmSubjectMap).get("idp"));
+                String nameId = SubjectUtils.getName((Subject) ((Map) realmSubjectMap).get("idp"),
+                        "You",
```

Comment:
This parameter is the default value for the display name.  In case the current subject does not have a display or name ID then You will be used.  The message gets displayed as "You logged out successfully.".